docs/tasks/argo.rst

```diff
@@ -0,0 +1,279 @@
+Leveraging ActionSet Creation using Argo Cron Workflows
+------------------------------------------------------------
+
+Argo Workflows enable us to schedule operations. In the Kanister project,
+Argo Cron Workflows will be used to automate the creation of ActionSets to
+backup an application at regular intervals.
+
+To summarize, ActionSets are CRDs that are used to backup or restore an application.
+The controller present in the Kanister namespace looks for ActionSets and
+executes them using the provided Blueprint or other ActionSet.
+
+In this tutorial you will schedule the creation of a backup ActionSet using
+Argo Cron Workflows.
+
+
+Prerequisites
+===============
+- Kubernetes 1.16+
+- Kanister controller version 0.80.0 installed in your cluster, let's assume in
```

Comment:
We should not use a constant version of `Kanister` here as that would get updated eventually. We can just use latest keyword Or point to the link to `Kanister` Install Guide.

Reply:
Done.